includes/class-freemius.php

```diff
@@ -14793,7 +14793,9 @@ function _submit_affiliate_application() {
             if ( ! $this->is_registered() ) {
                 // Opt in but don't track usage.
                 $next_page = $this->opt_in(
-                    false,

```

Comment:
@fajardoleo I think it is worth doing a sanity check here. Also from the form and flow it looks like if the user isn't opted in, we better get the email from the form.

```php
if ( ! $this->is_registered() ) {
  $email = isset( $affiliate['email'] ) ? $affiliate['email'] : '';

  if ( ! self::is_valid_email( $email ) ) {
    self::shoot_ajax_failure('Invalid email address');
  }

  // ...
}